tests/fixture/tmpnet/node_process.go

```diff
@@ -37,8 +37,9 @@ const (
 var (
 	AvalancheGoPluginDirEnvName = config.EnvVarName(config.EnvPrefix, config.PluginDirKey)
 
-	errNodeAlreadyRunning = errors.New("failed to start node: node is already running")
-	errNotRunning         = errors.New("node is not running")
+	errNodeAlreadyRunning   = errors.New("failed to start node: node is already running")
```

Comment:
nit I feel the `failed to start node: ` is unnecessary given the error `errNodeAlreadyRunning` is returned by the `NodeProcess.Start` method, and the caller of `NodeProcess.Start` should be the one adding the `failed to start node: %w` error wrapping?

```suggestion
	errNodeAlreadyRunning   = errors.New("node is already running")
```